29_2_hw_modified_dag.py

```diff
@@ -0,0 +1,102 @@
+import logging
+
+import pandas as pd
+import pendulum
+import requests
+from airflow import DAG
+from airflow.models import (
+    Param,
+    Variable,
+)
+from airflow.operators.python import PythonOperator
+from airflow.providers.postgres.hooks.postgres import PostgresHook
+from airflow.providers.postgres.operators.postgres import PostgresOperator
+
+
+def load_biogrid(params, ti):
+    biogrid_version = params['version']
+    if biogrid_version == 'latest':

```

Comment:
Now I get what you mean - this way it leads to a nasty errors. Imagine that you've loaded a data now, it was ingested into a db with a version 'latest', and in a few months new version was released. In our latest version of scxript it would not be working. + scientists usually need to know for their researches with which exactly version of data they work. So, it would be nice to parse a page and retrieve a version